modules/phase_field/src/vectorpostprocessors/FeatureVolumeVectorPostprocessor.C

```diff
@@ -33,7 +33,7 @@ validParams<FeatureVolumeVectorPostprocessor>()
                         "Set this Boolean if you wish to use an element based volume where"
                         " the dominant order parameter determines the feature that accumulates the "
                         "entire element volume");
-
+  params.addParam<bool>("output_centroids", false, "Set to true to output the feature centroids");
```

Comment:
Please remove unrelated changes (but out of curiosity, have you thought about periodicity?)